releases/release-1.16/release-notes-draft.md

```diff
@@ -213,154 +247,171 @@
 
 ## Notes From Multiple SIGs
 
-### SIG API Machinery, SIG CLI, SIG Node, and SIG Testing
+### SIG API Machinery, and SIG Testing
 
-- Fix kubelet errors in AArch64 with huge page sizes smaller than 1MiB ([#78495](https://github.com/kubernetes/kubernetes/pull/78495), [@odinuge](https://github.com/odinuge))
+- Fixes a bug in openapi published for custom resources using x-kubernetes-preserve-unknown-fields extensions, so that kubectl will allow sending unknown fields for that portion of the object. ([#79636](https://github.com/kubernetes/kubernetes/pull/79636), [@liggitt](https://github.com/liggitt))
+- A new client `k8s.io/client-go/metadata.Client` has been added for accessing objects generically. This client makes it easier to retrieve only the metadata (the `metadata` sub-section) from resources on the cluster in an efficient manner for use cases that deal with objects generically, like the garbage collector, quota, or the namespace controller. The client asks the server to return a `meta.k8s.io/v1 PartialObjectMetadata` object for list, get, delete, watch, and patch operations on both normal APIs and custom resources which can be encoded in protobuf for additional work. If the server does not yet support this API the client will gracefully fall back to JSON and transform the response objects into PartialObjectMetadata. ([#77819](https://github.com/kubernetes/kubernetes/pull/77819), [@smarterclayton](https://github.com/smarterclayton))
+
+### SIG API Machinery, SIG Apps, SIG Architecture, SIG Node, and SIG Scheduling
+
+- The deprecated node condition type 'OutOfDisk' has been removed. Use the 'DiskPressure' condition instead. ([#72420](https://github.com/kubernetes/kubernetes/pull/72420), [@Pingan2017](https://github.com/Pingan2017))
 
 ### SIG Cluster Lifecycle, and SIG GCP
 
 - Fixed a bug in kube-addon-manager's leader election logic that made all replicas active. ([#80575](https://github.com/kubernetes/kubernetes/pull/80575), [@mborsz](https://github.com/mborsz))
 
-### SIG CLI, SIG Cluster Lifecycle, and SIG Testing
+### SIG API Machinery, SIG Apps, SIG CLI, SIG Cloud Provider, SIG Cluster Lifecycle, SIG Network, SIG Node, and SIG Testing
 
-- E2E tests no longer add command line flags directly to the command line, test suites that want that need to be updated if they don't use HandleFlags
-  loading a -viper-config=e2e.yaml with suffix (introduced in 1.13) works again and now has a regression test ([#75593](https://github.com/kubernetes/kubernetes/pull/75593), [@pohly](https://github.com/pohly))
+- "NONE" ([#79392](https://github.com/kubernetes/kubernetes/pull/79392), [@Bowenislandsong](https://github.com/Bowenislandsong))
```

Comment:
@paulbouwer @onyiny-ang @kcmartin @cartyc Is anyone working on this one right now? Otherwise I could jump in for a quick fix.

Reply:
I believe @paulbouwer is working on / has a fix